packages/lib/server-only/2fa/verify-2fa-token.ts

```diff
@@ -17,6 +17,7 @@ export const verifyTwoFactorAuthenticationToken = async ({
   user,
   totpCode,
 }: VerifyTwoFactorAuthenticationTokenOptions) => {
+  // TODO: This is undefined and I can't figure out why.
```

Comment:
@Mythie

Reply:
This will be a build time thing, latest changes on main should deal with it I believe.

Reply:
Merged the latest changes on main and it is still undefined. It is wierd

Reply:
:S